brave-web-servlet-filter/src/main/java/com/github/kristofa/brave/servlet/BraveServletFilter.java

```diff
@@ -52,22 +55,32 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
             filterChain.doFilter(request, response);
         } else {
 
-            final StatusExposingServletResponse statusExposingServletResponse = new StatusExposingServletResponse((HttpServletResponse) response);
-            requestInterceptor.handle(new HttpServerRequestAdapter(new ServletHttpServerRequest((HttpServletRequest) request), spanNameProvider));
-
-            try {
-                filterChain.doFilter(request, statusExposingServletResponse);
-            } finally {
-                responseInterceptor.handle(new HttpServerResponseAdapter(new HttpResponse() {
-                    @Override
-                    public int getHttpStatusCode() {
-                        return statusExposingServletResponse.getStatus();
-                    }
-                }));
-            }
+        	final StatusExposingServletResponse statusExposingServletResponse = new StatusExposingServletResponse(
```

Comment:
it is very hard to tell what changed when formatting is mixed with code change. can you please revert the formatting change and ping back when only the essentials are shown? I'll review once that's done.